active_directory_ldap/plugin.spec.yaml

```
@@ -128,17 +141,19 @@ actions:
         description: Set this to true to disable the user account at creation
         required: true
         default: 'true'
+        example: 'true'
       user_principal_name:
         title: User Principal Name
-        description: The users principal name, e.g. user@example.com
+        description: The users principal name
         type: string
         required: true
+        example: user@example.com
       additional_parameters:
         title: Additional Parameters
-        description: 'Add additional user parameters in JSON format e.g. {''telephoneNumber'':
-          ''(617)555-1234''}'
+        description: 'Add additional user parameters in JSON format'
         type: object
         required: false
+        example: "{'telephoneNumber':'(617)555-1234'}"
```

There should be a valid JSON object
```suggestion
        example: '{"telephoneNumber":"(617)555-1234"}'
```